spanner_dbapi/cursor.py

```diff
@@ -104,7 +104,9 @@ def execute(self, sql, args=None):
             raise OperationalError(e.details if hasattr(e, "details") else e)
 
     def __handle_update(self, sql, params):
-        self._connection.in_transaction(self.__do_execute_update, sql, params)
+        self._connection.run_in_transaction(
```

Comment:
Where this `run_in_transaction` method is defined? Once you delete a dependency method from the `Connection` class, make sure its functionality is kept and/or transfered elsewhere, e.g., inside the `Cursor` class.